### PR TITLE
set-volume: add page

### PR DESCRIPTION
**Pull request by kabir1439**
[set-volume.md](https://github.com/tldr-pages/tldr/files/15433734/set-volume.md)

- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by spageektti** [comment]
Hello @kabir1439. Thanks for your contribution. 
Can you please rename the file from `Set-Volume.md` to `set-volume.md`?
And doesn't this command work only on Windows?

File: pages/common/Set-Volume.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Set-Volume
+
+> Sets or changes the file system label of an existing volume.
+> This command can only be used through Powershell.
+> More Information: <https://learn.microsoft.com/en-gb/powershell/module/storage/set-volume?view=windowsserver2022-ps>.
```
Comment (spageektti) [inline pages/common/Set-Volume.md:5]:
```suggestion
> More information: <https://learn.microsoft.com/powershell/module/storage/set-volume>.

```

File: pages/common/Set-Volume.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Set-Volume
+
+> Sets or changes the file system label of an existing volume.
+> This command can only be used through Powershell.
+> More Information: <https://learn.microsoft.com/en-gb/powershell/module/storage/set-volume?view=windowsserver2022-ps>.
+
+
```
Comment (spageektti) [inline pages/common/Set-Volume.md:7]:
```suggestion

```

File: pages/common/Set-Volume.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- Modify the properties of a volume using a volume object:
+
+`$volume = Get-Volume -DriveLetter "E"`
+`Set-Volume -InputObject $volume -NewFileSystemLabel "Backup"`
+
+- Specify the Data Deduplication mode for the volume:
+
+`Set-Volume -DriveLetter "D" -DedupMode Backup`
+
+
```
Comment (spageektti) [inline pages/common/Set-Volume.md:25]: Can you please remove these unnecessary blank lines?

File: pages/common/Set-Volume.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+`Set-Volume -DriveLetter "D" -NewFileSystemLabel "DataVolume"`
+
+- Change the file system label of a volume identified by the system label:
+
+`Set-Volume -FileSystemLabel "OldLabel" -NewFileSystemLabel "NewLabel"` 
+
+- Modify the properties of a volume using a volume object:
+
+`$volume = Get-Volume -DriveLetter "E"`
+`Set-Volume -InputObject $volume -NewFileSystemLabel "Backup"`
```
Comment (spageektti) [inline pages/common/Set-Volume.md:19]: We don't support examples with multiple lines.

**Review by spageektti — CHANGES_REQUESTED** [review]

**Comment by kabir1439** [comment]
[set-volume.md](https://github.com/tldr-pages/tldr/files/15433721/set-volume.md)

**Pull request closed without merging**